src/sage/groups/braid.py

```diff
@@ -1447,6 +1448,52 @@ def annular_khovanov_homology(self, qagrad=None, ring=IntegerRing()):
             return {qa: C[qa].homology() for qa in C}
         return self.annular_khovanov_complex(qagrad, ring).homology()
 
+    def left_normal_form(self, algorithm='artin'):
+        r"""
+        Return the left normal form of the braid.
+        
+        INPUT:
+        
+        - `algorithm` : string (default: ``artin``). For `artin` the general
+          method for Artin group is used. For `libbraiding`, the algorithm
+          uses the package `libbraiding`.
```

Comment:
```suggestion
        - ``algorithm`` -- string (default: ``'artin'``); must be one of the following:

          * ``'artin'`` -- the general method for Artin groups is used
          * ``'libbraiding'`` -- the algorithm uses the ``libbraiding`` package
```
Why also a default of `artin` instead of `libbraiding` since it is faster? I was only suggesting keeping both available, but you can certainly change the default.